code/modules/mob/living/simple_animal/hostile/mining_mobs/polarbear.dm

```diff
@@ -59,6 +59,12 @@
 	pull_force = PULL_FORCE_DEFAULT
 	return ..()
 
+/mob/living/simple_animal/hostile/asteroid/polarbear/revive(full_heal = FALSE, admin_revive = FALSE)
+	. = ..()
+	move_force = initial(move_force)
+	move_resist = initial(move_resist)
+	pull_force = initial(pull_force)
```

Comment:
won't this code run regardless of whether or not the revive attempt was successful?

Reply:
It will. Thanks for pointing this out.